narwhal/source_code_sharing/_liferay_common.sh

```diff
@@ -0,0 +1,390 @@
+#!/bin/bash
+
+#
+# Do NOT edit this file unless you are editing this file in the root directory
+# of the the liferay-docker repository. Edit that file, and then fork it to the
+# repository where it is used.
+#
+
+function lc_background_run {
```

Comment:
@lucasmiranda0, why are we reintroducing a new `_liferay_common.sh`? Can't we use the one we already have?